src/providers/pubsub.ts

```diff
@@ -98,6 +98,11 @@ export class TopicBuilder {
   }
 }
 
+/**
+ * The Google Cloud Pub/Sub schedule builder.
```

Comment:
I hesitate to say `Google Cloud Pub/Sub schedule builder.` here, because that sounds to me like something that is built into pubsub. However, this is really just the Cloud Scheduler job that we create to run scheduled functions with.

Maybe this could be something like:
```
The schedule builder for scheduled functions. This describes the Cloud Scheduler job that will be deployed to trigger a scheduled function at the provided frequency.
```

Reply:
I know . . . I think the reason this was overlooked was exactly that: we thought it was a class outside of CF3.

But here it is -- sourced from providers/pubsub.ts and generated along with the other files.

Joe do you think we should we actually be *hiding/omitting* this class from the reference docs instead of trying to add it and flesh it out?

Reply:
We should definitely be documenting it as a part of the firebase-functions reference docs. What about:

"The builder for scheduled functions, which are powered by Pub/Sub and Cloud Scheduler."

Reply:
See what you think about this amalgam of all feedback.  Snuck in a link to the guide.

Reply:
This version SGTM!